service/src/main/java/bio/terra/workspace/app/controller/ReferencedGcpResourceController.java

```diff
@@ -476,11 +481,31 @@ public ResponseEntity<ApiDataRepoSnapshotResource> createDataRepoSnapshotReferen
             .snapshotId(body.getSnapshot().getSnapshot())
             .build();
 
-    ReferencedDataRepoSnapshotResource referenceResource =
-        referencedResourceService
-            .createReferenceResource(resource, userRequest)
-            .castByEnum(WsmResourceType.REFERENCED_ANY_DATA_REPO_SNAPSHOT);
-    return new ResponseEntity<>(referenceResource.toApiResource(), HttpStatus.OK);
+    var linkPoliciesResults =
```

Comment:
This is the entry point for the change to link policies to a snapshot. Start here to follow the new feature.